koku/api/migrations/0018_auto_20200326_0102.py

```diff
@@ -7,16 +7,7 @@
 
 def reload_cost_model_metric_map(apps, schema_editor):
     """Load cost model metrics."""
-    CostModelMetricsMap = apps.get_model("api", "CostModelMetricsMap")
-    CostModelMetricsMap.objects.all().delete()
-
-    data = pkgutil.get_data("api", "metrics/data/cost_models_metric_map.json")
-
-    data = json.loads(data)
-
-    for entry in data:
-        map = CostModelMetricsMap(**entry)
-        map.save()
+    pass
```

Comment:
@adberglund  Can you take a look, similar to the note above.

Why not just remove both the method and the operation call:
https://github.com/project-koku/koku/pull/1986/commits/f1b23c08e09f91f7630f82e7fbacbc78b6aeeda9#diff-3dee2e30ad652d9aba5aa0b9f1427813R17

Reply:
updated @chambridge.

I removed the method and operation call.